includes/class-wp-job-manager.php

```diff
@@ -175,6 +175,8 @@ public function include_template_functions() {
 	 */
 	public function rest_init() {
 		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/class-wp-job-manager-rest-api.php';
+		include_once JOB_MANAGER_PLUGIN_DIR . '/includes/class-wp-job-manager-promoted-jobs-api.php';
```

Comment:
I wonder if it would be better to initialize it as part of the `WP_Job_Manager_Promoted_Jobs` class. Not sure what is the standard we're using for other endpoints, if we have something defined already though.

I also just noticed that `WP_Job_Manager_Promoted_Jobs` is being included in the `includes/admin/class-wp-job-manager-admin.php`. Maybe we should include it in a more generic class since we'll have things not related to the admin?

Reply:
What if we include `WP_Job_Manager_Promoted_Jobs` in `class-wp-job-manager.php` instead, then we can load the rest API from `WP_Job_Manager_Promoted_Jobs`?  

For the plugin side, what will it have that's not related to admin?

Reply:
> What if we include `WP_Job_Manager_Promoted_Jobs` in `class-wp-job-manager.php`.

I think it makes sense!

> For the plugin side, what will it have that's not related to admin?

It would be the admin part and the REST APIs, for this first iteration of the project.

Reply:
Done in https://github.com/Automattic/WP-Job-Manager/pull/2469/commits/dcbf48a0a3346bdadbf31095574b7f2e3fa3c76b